packages/kbn-legacy-logging/src/utils/get_payload_size.test.ts

```diff
@@ -72,6 +78,11 @@ describe('getPayloadSize', () => {
       const result = getResponsePayloadBytes(payload);
       expect(result).toBe(JSON.stringify(payload).length);
     });
+
+    test('returns undefined when source is not plain object', () => {
+      const result = getResponsePayloadBytes([1, 2, 3]);
```

Comment:
that seems to be a valid use-case when you want to respond with an array of objects. isn't it? @lukeelmers